legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverageAnalyticsTest.pure

```diff
@@ -0,0 +1,236 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import meta::analytics::mapping::modelCoverage::*;
```

Comment:
Ok, it's kinda nice that we have tests here, but I'm not sure if we want `relational` test to be here. Also, the test you have here won't run since you don't call it